crates/fj-kernel/src/objects/full/face.rs

```diff
@@ -35,9 +35,7 @@ use crate::{
 #[derive(Clone, Debug, Eq, PartialEq, Hash, Ord, PartialOrd)]
 pub struct Face {
     surface: Handle<Surface>,
-    exterior: Handle<Cycle>,
-    interiors: Vec<Handle<Cycle>>,
-    color: Option<Color>,
+    region: Region,
```

Comment:
Should this be a `Region` or `Handle<Region>`?

Reply:
That question is basically the same as asking, is `Region` an object (i.e. stored in centralized storage and referred to via `Handle`) or is it just a normal struct. It should be an object, if it needs an identity ([documented here](https://docs.rs/fj-kernel/0.44.0/fj_kernel/storage/struct.Handle.html#equality-and-identity)), which can be relevant for validation (see https://github.com/hannobraun/Fornjot/issues/1689, for example).

The the answer will probably become obvious at some point. For now, I don't know, but I can't think of a reason why it needs to be an object. I think that means it would be best to make it a normal struct, as that's less complex. If it needs to become an object at some point, we can make that happen then.

If it's not an object, that means it should probably live in [`geometry`](https://docs.rs/fj-kernel/0.44.0/fj_kernel/geometry/index.html), rather than within the `objects` module.

Good catch, by the way! I hadn't thought about this before, and had just assumed that it needs to be an object.